src/content/api/cli.md

```diff
@@ -210,10 +210,16 @@ This is the lookup priority in increasing order
 
 W> Note that Command Line Interface has a higher precedence for the arguments you use it with than your configuration file. For instance, if you pass [`--mode="production"`](/configuration/mode/#usage) to webpack CLI and your configuration file uses `development`, `production` will be used.
 
+### `help`
```

Comment:
```suggestion
### help
```
Can you rebase the pull request? And let's avoid inline code in heading, it makes the page messy in my opinion.

Reply:
resolved